src/openrct2/platform/Platform.Win32.cpp

```diff
@@ -90,6 +90,92 @@ namespace Platform
         return String::ToUtf8(result);
     }
 
+    typedef LONG NTSTATUS, *PNTSTATUS;
+#    define STATUS_SUCCESS (0x00000000)
+
+    typedef NTSTATUS(WINAPI* RtlGetVersionPtr)(PRTL_OSVERSIONINFOW);
+
+    RTL_OSVERSIONINFOW GetRealOSVersion()
+    {
+        HMODULE hMod = ::GetModuleHandleW(L"ntdll.dll");
+        if (hMod)
+        {
+            RtlGetVersionPtr fxPtr = (RtlGetVersionPtr)::GetProcAddress(hMod, "RtlGetVersion");
+            if (fxPtr != nullptr)
+            {
+                RTL_OSVERSIONINFOW rovi = { 0 };
+                rovi.dwOSVersionInfoSize = sizeof(rovi);
+                if (STATUS_SUCCESS == fxPtr(&rovi))
+                {
+                    return rovi;
+                }
+            }
+        }
+        RTL_OSVERSIONINFOW rovi = { 0 };
+        return rovi;
+    }
+
+    std::string GetOsName()
+    {
+        std::string output = "";
+        SYSTEM_INFO si;
+        GetNativeSystemInfo(&si);
+
+        auto VN = GetRealOSVersion();
+
+        if (VN.dwMajorVersion == 10 && VN.dwBuildNumber >= 22000)
+        {
+            output += "Windows 11";
+        }
+        else if (VN.dwMajorVersion == 10 && VN.dwBuildNumber < 22000)
+        {
+            output += "Windows 10";
+        }
+        else if (VN.dwMajorVersion == 6 && VN.dwMinorVersion == 3)
+        {
+            output += "Windows 8.1";
+        }
+        else if (VN.dwMajorVersion == 6 && VN.dwMinorVersion == 2)
+        {
+            output += "Windows 8";
+        }
+        else if (VN.dwMajorVersion == 6 && VN.dwMinorVersion == 1)
+        {
+            output += "Windows 7";
+        }
+        else if (VN.dwMajorVersion == 6 && VN.dwMinorVersion == 0)
+        {
+            output += "Windows Vista";
+        }
+        else if (VN.dwMajorVersion == 5 && VN.dwMinorVersion >= 1)
+        {
+            output += "Windows XP";
+        }
+        else if (VN.dwMajorVersion == 5 && VN.dwMinorVersion == 0)
+        {
+            output += "Windows 2000";
+        }
+        else
+        {
+            output += "Windows";
+        }
+
+        if (si.wProcessorArchitecture == 6 || si.wProcessorArchitecture == 9)
```

Comment:
Use named values such as `PROCESSOR_ARCHITECTURE_AMD64` and provide that information in report as well: `AMD64` or `ARM64` instead of just `64 bit`

Reply:
Just directly added architecture.

Reply:
Doesn't seem resolved yet - missing a commit?